src/components/program.jsx

```diff
@@ -165,8 +165,8 @@ function Program ({ eventsList, tagsList, themesList }) {
             <Collapse in={open} className='bg-light border p-3 rounded-sm'>
               <div id='searchEvent' className='searchEvent mb-3'>
                 <SearchBox value={searchText} onChange={handleSearch} />
-                <div className='mb-3'>
-                  {themes && (
+                {themes.length > 1 && themes && (
```

Comment:
This should be `themes && themes.length > 0`, i.e. first check if `themes` is set and then check the length. And the length should be `> 0` if there are any themes.